buildprocess/generateDocs.ts

```diff
@@ -291,12 +299,12 @@ export default async function generateDocs() {
 
   // Add entries for all the catalog item/group/function/reference types to type details subsection in mkdocs.yml
   const connectingToDataSection = mkDocsConfig.nav
-    .map((section:any) => section["Connecting to Data"])
-    .filter((x:any) => x !== undefined)[0];
+    .map((section: any) => section["Connecting to Data"])
+    .filter((x: any) => x !== undefined)[0];
   const typeDetailsSubSection =
     connectingToDataSection &&
     connectingToDataSection.find(
-      (subSection:any) => "Catalog Type Details" in subSection
+      (subSection: any) => "Catalog Type Details" in subSection
     );
   if (typeDetailsSubSection === undefined) {
     throw new Error(
```

Comment:
I guess all of these are just prettier changes.

Reply:
yep!